lib/src/views/feed/components/post/post_actions.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:feed_sx/src/utils/constants/ui_constants.dart';
+import 'package:feed_sx/src/utils/constants/assets_constants.dart';
+import 'package:feed_sx/src/utils/constants/string_constants.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
```

Comment:
The package I am talking about

Reply:
> Couple of thoughts -
> 
> 1. The assets are all svg files, this required you to use a plugin in the buttons (specified in further comments), were they not available in the builtin icons class?
> 2. Check the directories, we not necessarily have so many excess files and directories. Maybe the asset and string constants can be in the same file since you have to import both anyway.
> 3. Specific comments left in each file.

1. Explained the thought process in the comment.
2. It is always beneficial to separate the code into different files. For example, if we put all the constants like `assets_constants`, `string_constants` in a single file,  even when a file needs only `string_constants` and does not use the `assets_constants`, dart will spend time parsing the `assets_constants` as well. Also, it increases the readability of the code.

Reply:
Okay, makes sense!